src/Aspire.Dashboard/Components/Controls/PlotlyChart.razor.cs

```diff
@@ -431,12 +431,23 @@ private async Task UpdateChart(bool tickUpdate, DateTime inProgressDataTime)
 
         if (!tickUpdate)
         {
+            // The chart mostly shows numbers but some localization is needed for displaying time ticks.
+            var is24Hour = DateTimeFormatInfo.CurrentInfo.LongTimePattern.StartsWith("HH", StringComparison.OrdinalIgnoreCase);
```

Comment:
This can't be case insensitive. `hh` is 12-hour format. You want either `H` or `HH` for 24-hour format.

```suggestion
            var is24Hour = DateTimeFormatInfo.CurrentInfo.LongTimePattern.StartsWith("H", StringComparison.Ordinal);
```

Reply:
It worked on my culture because 24 is HH and 12 hour is just h